File: aluguel-core/src/main/java/com/nortetec/locatario/LocatarioServiceImpl.java
package com.nortetec.locatario;

import org.springframework.stereotype.Service;

import java.util.List;

@Service
public class LocatarioServiceImpl implements LocatarioService {

    private final LocatarioRepository repository;

    public LocatarioServiceImpl(LocatarioRepository repository) {
        this.repository = repository;
    }

    @Override
    public List<Locatario> listaLocatario() {
        return repository.findAll();
    }

}

File: src/main/kotlin/com/nortetec/locatario/LocatarioServiceImpl.kt
package com.nortetec.locatario

import org.springframework.stereotype.Service

@Service
class LocatarioServiceImpl(val repository: LocatarioRepository) : LocatarioService {
    override fun hello(): String {
        return "hello service"
    }

    override fun listaLocatario(): List<Locatario> {
        return repository.findAll()
    }
}